include/ccf/service/tables/modules.h

```diff
@@ -24,5 +25,7 @@ namespace ccf
       "public:ccf.gov.modules_quickjs_bytecode";
     static constexpr auto MODULES_QUICKJS_VERSION =
       "public:ccf.gov.modules_quickjs_version";
+    static constexpr auto INTERPRETER_FLUSH =
+      "public:ccf.gov.interpreter.flush";
```

Comment:
This new table should be documented [here](https://microsoft.github.io/CCF/main/audit/builtin_maps.html).

Reply:
Good point, this wasn't caught by the automation because it's run at the end of the full suite, which doesn't run JS code right now.